temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
```

Comment:
Import the `json` module at the beginning of the file to ensure that `json.dumps({})` can be used within the `reset_files` function. [important]